src/sentry/static/sentry/app/views/settings/organizationMembers/organizationMembersWrapper.tsx

```diff
@@ -0,0 +1,194 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+import {openInviteMembersModal} from 'app/actionCreators/modal';
+import {Panel} from 'app/components/panels';
+import {t} from 'app/locale';
+import AsyncView from 'app/views/asyncView';
+import Badge from 'app/components/badge';
+import Button from 'app/components/button';
+import InlineSvg from 'app/components/inlineSvg';
+import ListLink from 'app/components/links/listLink';
+import NavTabs from 'app/components/navTabs';
+import routeTitleGen from 'app/utils/routeTitle';
+import SentryTypes from 'app/sentryTypes';
+import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
+import space from 'app/styles/space';
+
+type Props = AsyncView['props'] & {
+  children?: any;
+};
+
+class OrganizationMembersWrapper extends AsyncView<Props, AsyncView['state']> {
```

Comment:
Do you actually have to pass the `AsyncView['state']` here? I think it just works if you leave it blank since it has a default right?